tests/grpcio-wheel/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib, poetry2nix, python3, pkgs, runCommand }:
+let
+  inherit (pkgs.stdenv) isLinux;
+  env = poetry2nix.mkPoetryEnv {
+    python = python3;
+    pyproject = ./pyproject.toml;
+    poetrylock = ./poetry.lock;
+    preferWheels = false;
```

Comment:
Are any of the above needed? They are all default values, right? (Let me know if this sort of comment is too nitpicky; as a developer I'm used to getting rid of any duplication of defaults.)